src-script/pack-apack-mac.js

```diff
@@ -5,31 +5,48 @@ const sevenBin = require('7zip-bin')
 const pathTo7zip = sevenBin.path7za
 
 exports.default = async function (buildResult) {
-  buildResult?.artifactPaths?.forEach((element) => {
+  for (const element of buildResult?.artifactPaths || []) {
     if (
       (element.includes('mac') ||
         element.includes('win') ||
         element.includes('linux')) &&
       element.endsWith('.zip')
     ) {
-      const myStream = Seven.add(
-        element,
-        path.join(buildResult.outDir, '../apack.json'),
-        {
-          $progress: true,
-          $bin: pathTo7zip
-        }
-      )
-      // myStream.on('end', function () {
-      //   console.log(myStream.info)
-      // })
+      // Add apack.json first
+      await new Promise((resolve, reject) => {
```

Comment:
Why do we need to await new promise for both apack.json and zap.png?

Reply:
if I don't put it in a promise chain only one file shows up